docs/reference/nuget-config-file.md

```diff
@@ -35,6 +35,7 @@ Contains miscellaneous configuration settings, which can be set using the [`nuge
 | repositoryPath (`packages.config` only) | The location in which to install NuGet packages instead of the default `$(Solutiondir)/packages` folder. A relative path can be used in project-specific `nuget.config` files. |
 | defaultPushSource | Identifies the URL or path of the package source that should be used as the default if no other package sources are found for an operation. |
 | http_proxy http_proxy.user http_proxy.password no_proxy | Proxy settings to use when connecting to package sources; `http_proxy` should be in the format `http://<username>:<password>@<domain>`. Passwords are encrypted and cannot be added manually. For `no_proxy`, the value is a comma-separated list of domains the bypass the proxy server. You can alternately use the http_proxy and no_proxy environment variables for those values. For additional details, see [NuGet proxy settings](http://skolima.blogspot.com/2012/07/nuget-proxy-settings.html) (skolima.blogspot.com). |
+| maxHttpRequestsPerSource (projects using PackageReference only)| Controls the maximum number of parallel requests sent from NuGet to every package source for package dependency resolution and downloads. The default value is `256` which is derived from `HttpClientHandler.MaxConnectionsPerServer` property. This setting has no impact on `.NET Core` for `Mac OS` because the throttling limit is set to `16` to avoid too many open files error. `nuget.exe` sets `ServicePointManager.DefaultConnectionLimit` property to `64` on `Windows` and to `1` on `Mono` to control maximum number of concurrent connections allowed for every package source. Hence configuring `maxHttpRequestsPerSource` property to a value less than `64` could impact performance of `nuget.exe` on `Windows`. |
```

Comment:
How confident are we that it's PackageReference only? 
I was under the impression that it's both.

Reply:
Great question Nikolche. You are correct. After reading [wiki](https://github.com/NuGet/Home/wiki/Config-Max-Http-Request-In-NuGet) especially "Package.config use maxDegreeOfParallelism  which is set to Environment.ProcessorCount by default for [dependency fetching](https://github.com/NuGet/NuGet.Client/blob/f82a7ac6b54719225cda5f3317e3c31d5843279d/src/NuGet.Core/NuGet.PackageManagement/Resolution/ResolverGather.cs#L42) and [package download & extraction](https://github.com/NuGet/NuGet.Client/blob/9e1e8b52818b4bc5dbd7e8b6049fcdf15cad04b2/src/NuGet.Core/NuGet.PackageManagement/IDE/PackageRestoreManager.cs#L393)", I thought this setting is only applicable to `PR` projects which seems to be incorrect. I pushed a new commit with updated text. Please let me know what you think. I am happy to learn more if the content is incorrect.